### ISSUE TITLE
Support for babel-plugin-react-intl-auto

### ISSUE THREAD

**Issue by jdcrensh** [avaragado/riw#7]
It would be great if this tool supported [`babel-plugin-react-intl-auto`](https://github.com/akameco/babel-plugin-react-intl-auto). It allows `id`s to be autogenerated based on the file's path.

I installed the plugin and added it to my `.babelrc`, but it doesn't look like `riw` is picking up the plugin.

`.babelrc`
```json
{
  "presets": ["react-app"],
  "plugins": [
    ["react-intl-auto", {
      "removePrefix": "src/",
      "filebase": false
    }]
  ]
}
```

I dug in a little bit, and it looks like `riw` uses `babel-plugin-react-intl` explicitly: https://github.com/avaragado/riw/blob/d48782409a29e4181957bcf08003f54a8c9119bc/src/lib/riw/app/translate/armdFromSourceFabs.js#L17

Would it be possible to include additional babel plugins, or am I missing something?

**Comment by avaragado** [comment]
Thanks for the report! 

You're right, at the moment the code runs a babel transform explicitly using `babel-plugin-react-intl`. I'm not familiar with `babel-plugin-react-intl-auto` but it may be possible to get them to play nicely together. If I get a chance in the next couple of days I'll take a look.

**Comment by jdcrensh** [comment]
I was able to add `babel-plugin-react-intl-auto` to `riw` without breaking any tests (it has to be run before `babel-plugin-react-intl`) and it worked as expected. However, making the plugin user-configurable through `riw.config.js` was a little beyond me, since the `Config` object isn't passed to that function.

**Comment by avaragado** [comment]
Good to know it works! Would you be happy with a riw config setting that defines any extra plugins to run? I'm pretty sure that's doable, once I get my head back into the code.

**Comment by jdcrensh** [comment]
Ideally, it'd be preferable if the plugin was blackboxed just as `babel-plugin-react-intl` is, since there'd be no other use for it other than passing off to `riw`. We'd rather not have babel dependencies in our `create-react-app` apps 😉 

Maybe it could default the config to `reactIntlAuto:{"removePrefix": "src/","filebase": false}`, or not use the plugin if `false`?

**Comment by avaragado** [comment]
@jdcrensh Released in 0.7.0.

**Comment by jdcrensh** [comment]
You're awesome @avaragado! Updated and it's working great 👍
